src/lib/frontend/components/Image.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import {dev} from '$app/environment'
+
+    export let name: string
+    export let image: string
+    export let classes: string
+    export let width = 160
+
+    function optimize(src: string, widths = [160, 640], quality = 90) {
+        if (dev) return src
+
+        return widths
+            .slice()
+            .sort((a, b) => a - b)
+            .map((width, i) => {
+                const url = `/_vercel/image?url=${encodeURIComponent(src)}&w=${width}&q=${quality}`
+                const descriptor = i < widths.length - 1 ? ` ${width}w` : ''
+                return url + descriptor
+            })
+            .join(', ')
+    }
+</script>
+<img alt={name}
+     class="{classes}"
+     src={optimize(image, [width], 90)}
```

Comment:
shouldn't be src instead of image? img tag  has a src props. we could have the same pattern with our Image tag

Reply:
Naming them the same would lead to believe that they are interchangeable but they are not yet, we only support remote url. I will rename it to remoteUrl.